### PR TITLE
Universal Profiling: add package for symbolizer

### PR DESCRIPTION
**Pull request by florianl**
## What does this PR do?

Add a package for Universal Profiling Symbolizer

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/universal_profiling_symbolizer/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+The minimum supported versions are:
+
+* PHP: >= 7.3
+* Python: >= 3.6
+* JVM/JDK: >= 7
+* V8: >= 8.1.0
+* Ruby: >= 2.5
+* Perl: >= 5.28
+
+[Learn more](https://elastic.github.io/universal-profiling-documentation)
```
Comment (inge4pres) [inline packages/universal_profiling_symbolizer/docs/README.md:36]: You may want to use the elastic guides link

```suggestion
[Learn more](https://www.elastic.co/guide/en/observability/current/universal-profiling.html)
```

File: packages/universal_profiling_symbolizer/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - size: 48x46
+    src: /img/profiler-logo-color-48px.svg
+    type: image/svg+xml
+    title: Universal Profiling Logo
+policy_templates:
+  - name: Universal Profiling Symbolizer
+    title: Universal Profiling Symbolization Service
+    description: Interact with Universal Profiling.
+    inputs:
+      - title: Collect Universal Profiling traces
```
Comment (inge4pres) [inline packages/universal_profiling_symbolizer/manifest.yml:20]: This seems related to the collector, but we do have an input for symbolization alone.

File: packages/universal_profiling_symbolizer/manifest.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        description: Collect Universal Profiling traces
```
Comment (inge4pres) [inline packages/universal_profiling_symbolizer/manifest.yml:21]: Rather than `traces`, we can say `debug symbols`.

**Review by kpollich — APPROVED** [review]
LGTM 🚀

File: packages/universal_profiling_symbolizer/manifest.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    inputs:
+      - title: Universal Profiling Symbolizer
+        description: Enhance Universal Profiling with additional symbol information
+        type: pf-elastic-symbolizer
+        vars:
+          - name: hack
+            type: bool
+            title: hack to workaround a known issue
+            description: |
+              Check out elastic/kibana/issues/155064 for the reason of this hack.
```
Comment (kpollich) [inline packages/universal_profiling_symbolizer/manifest.yml:28]: Should be able to remove this once https://github.com/elastic/kibana/pull/155079 lands

File: packages/universal_profiling_symbolizer/changelog.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: 8.8.0-preview
+  changes:
+    - description: Initial draft of the package
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/1
```
Comment (inge4pres) [inline packages/universal_profiling_symbolizer/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/5879
```

This one should be updated

**Review by inge4pres — APPROVED** [review]
Minor non-blocking comment.
Thanks for creating this 🙏🏼 
As soon as it is merged, I think we can test it in Cloud.

File: packages/universal_profiling_symbolizer/manifest.yml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+icons:
+  - size: 48x46
+    src: /img/profiler-logo-color-48px.svg
+    type: image/svg+xml
+    title: Universal Profiling Logo
+policy_templates:
+  - name: universal_profiling_symbolizer
+    title: Universal Profiling Symbolization Service
+    description: Interact with Universal Profiling.
+    inputs:
```
Comment (inge4pres) [inline packages/universal_profiling_symbolizer/manifest.yml:19]: @joshdover is it here we need to add the data stream?

File: packages/universal_profiling_symbolizer/manifest.yml [hunk 7]
```diff
@@ -0,0 +1,32 @@
+name: profiler_symbolizer
+title: Universal Profiling Symbolizer
+version: 8.8.0-preview
+categories: ["elastic_stack", "monitoring"]
+description: Fleet-wide, whole-system, continuous profiling with zero instrumentation.
+conditions:
+  kibana.version: ^8.8.0
+  elastic.subscription: basic
+format_version: 1.1.0
```
Comment (inge4pres) [inline packages/universal_profiling_symbolizer/manifest.yml:9]: This needs to be upgraded to 2.7.0 to support the new data stream, once we add it

**Pull request merged**